### PR TITLE
Homepage news section

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Make space (Update solutions)
- [x] Add basic news section with fake content (in yml's)
- [x] Update design
- [x] Add real contents (initial news items, solutions title)
  - [x] Dutch
  - [x] German
  - [x] English
- [x] Update content

### CODE REVIEW

File: locales/de.yml [hunk 1]
```diff
@@ -315,7 +315,7 @@ de:
   press:
     heading: "Presse"
   products:
-    title: "Produkte, auf die wir stolz sind."
+    title: "Produkte, auf die wir stolz sind"
     capp:
       learn: "Bilden"
       develop: "Entwickeln"
```
Comment (RonaldDijkstra) [inline locales/de.yml]: Just removed the dot.

**Comment by RonaldDijkstra** [comment]
@sn3p ready for review

File: source/localizable/home/_news.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          <div class="text">
+            <p>
+              <%= news_item.meta %>
+            </p>
+            <h3>
+              <%= news_item.title %>
+            </h3>
+          </div>
+        <% end %>
+        <%= nav_link_to "", news_item.link, class: "mobile-trigger" %>
```
Comment (fatboypunk) [inline source/localizable/home/_news.erb]: This gets a title from i18n?

Comment (sn3p) [inline source/localizable/home/_news.erb]: Do we need a second link here?

Comment (RonaldDijkstra) [reply source/localizable/home/_news.erb]: It's for the arrow, but I'll remove that.

**Comment by RonaldDijkstra** [comment]
It's not perfect but this will do for now. 
I just made the title shorter and used an extra div to put flex on. 

@fatboypunk I used news_item.title for the title on the anchors, thanks for that feedback.

I want this deployed and then next week make it perfect. Can that be done? @sn3p

File: source/stylesheets/_news.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+      &:hover {
+        img {
+          transform: scale(1.1);
+        }
+      }
+
+      .text {
+        flex: 1;
+        padding-left: 1em;
```
Comment (sn3p) [inline source/stylesheets/_news.scss]: Maybe add a bit of padding to the right as well?

screen shot 2017-09-21 at 21 20 07

**Pull request merged**